### PR TITLE
switch to lodash fp

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

File: src/components/AuthContainer.js [hunk 1]
```diff
@@ -26,11 +26,11 @@ async function createClusters() {
   const [billingProjects, clusters] = await Promise.all(
     [Rawls.listBillingProjects(), Leo.clustersList()])
   const projectsWithoutClusters = _.difference(
```
Comment (panentheos) [inline src/components/AuthContainer.js:28]: `difference` doesn't have flipped arguments

File: src/libs/utils.js [hunk 2]
```diff
@@ -10,8 +10,8 @@ const subscribable = () => {
       subscribers = _.union(subscribers, [fn])
     },
     unsubscribe: fn => {
-      console.assert(_.includes(subscribers, fn), 'Function is not subscribed')
-      subscribers = _.difference(subscribers, [fn])
+      console.assert(_.includes(fn, subscribers), 'Function is not subscribed')
+      subscribers = _.difference([fn], subscribers)
```
Comment (panentheos) [inline src/libs/utils.js]: Another `difference`

File: src/components/AuthContainer.js [hunk 3]
```diff
@@ -26,11 +26,11 @@ async function createClusters() {
...
-    _.uniq(_.map(billingProjects, 'projectName')), // in case of being both a user and an admin of a project
     _.map(
-      _.filter(clusters, { creator: userProfile.getEmail() }),
-      'googleProject'
-    )
+      'googleProject',
+      _.filter(clusters, { creator: userProfile.getEmail() })
```
Comment (panentheos) [inline src/components/AuthContainer.js]: This `filter` needs to be flipped

**Review by panentheos — APPROVED** [review]
Looks good overall, but see comments.

**Review by MattPutnam — APPROVED** [review]

**Pull request merged**